files/zh-cn/web/css/gradient/linear-gradient/index.html

```diff
@@ -40,7 +40,7 @@ <h2 id="线形渐变的构成"><strong>线形渐变的构成</strong></h2>
 
 <p>线性渐变由一个轴 (梯度线) 定义，其上的每个点具有两种或多种的颜色，且轴上的每个点都具有独立的颜色。为了构建出平滑的渐变，<code>linear-gradient()</code> 函数构建一系列垂直于渐变线的着色线，每一条着色线的颜色则取决于与之垂直相交的渐变线上的色点</p>
 
-<p><img alt="linear-gradient.png" src="/files/3537/linear-gradient.png" style="float: left; height: 383px; width: 309px;">渐变线由包含渐变图形的容器的中心点和一个角度来定义的。渐变线上的颜色值是由不同的点来定义，包括起始点，终点，以及两者之间的可选的中间点（中间点可以有多个）。</p>
+<p><img alt="linear-gradient.png" src="/zh-cn/web/css/gradient/linear-gradient/linear-gradient.png" style="height: 383px; width: 309px;">渐变线由包含渐变图形的容器的中心点和一个角度来定义的。渐变线上的颜色值是由不同的点来定义，包括起始点，终点，以及两者之间的可选的中间点（中间点可以有多个）。</p>
```

Comment:
Just use `linear-gradient.png` and **remove the `linear-gradient.png` file** in this commit ([mdn/yari](https://github.com/mdn/yari) will fallback to `en-US` to fetch this image). Additionally, please help to remove the `style` attribute:

```html
<img alt="linear-gradient.png" src="linear-gradient.png">
```